static/app/views/settings/organizationMembers/inviteRequestRow.tsx

```diff
@@ -119,24 +138,18 @@ const InviteRequestRow = ({
             size="small"
             busy={inviteRequestBusy[inviteRequest.id]}
             title={
-              roleDisallowed
+              roleDisallowed && canApprove
```

Comment:
@Neo-Zhixing We probably need two different titles here. One if `canApprove` is true but `roleDisallowed` is false (the current message) and another if `canApprove` is false. In the latter case, the user can't approve users at all (the bit about selecting a different role isn't correct in this case).

Reply:
Added a message when `canApprove` is false.